### ISSUE TITLE
 Set a value through javascript, without setValue()

### ISSUE THREAD

**Issue by augustomauro** [Choices-js/Choices#1018]
I need to set the values ​​as normal via document.getElementById('select').value = 2 and have it hit like a normal change event and have the Choice.js plugin recognize it perfectly and display it. I observe that the original select loses all the <option> and there is only one value left that I don't know why it leaves it or under what criteria, but if there are no other options left in the select I will never be able to assign a value through the method mentioned. I can't use setValue() on the instance (which I store one by one in an array of choices objects) because I need to maintain the normal assignment of values ​​in each select, as if the component didn't exist.
The way to instantiate each choice is:

document.querySelectorAll('select').forEach((select) = > {
                            selectChoices.push(new Choices(select, {
                                allowHTML: false,
                                silent: true,
                                placeholder: true,
                                placeholderValue: '',
                                searchPlaceholderValue: 'Ingrese texto',
                                loadingText: 'Cargando...',
                                noResultsText: 'No se encontraron resultados',
                                noChoicesText: 'No hay opciones para elegir',
                                itemSelectText: '',
                                searchFields: ['label'],    // por defecto ['label','value']. Para buscar solo sobre las etiquetas innerText y no los values del option
                                // searchFloor: 2,
                                shouldSort: false,
                                // Para mejorar la busqueda de matcheo (ref: https://github.com/Choices-js/Choices/issues/135)
                                fuseOptions: {
                                    threshold: 0.1,
                                    distance: 1000
                                }
                            })
});
Thanks !!

**Comment by pecuchet** [comment]
@augustomauro, yes, that would also be a better solution for #277 @jshjohnson, doesn't Choices.js listen for `change` on the `<select>` element?
